docs/oadp_cheat_sheet.md

```
@@ -61,66 +61,67 @@ oc explain dataprotectionapplications.oadp.openshift.io
+access (not just Velero's restic subcommands), you can still alias it:
 ```
-alias velero='oc -n openshift-adp exec deployment/velero -c velero -it -- ./velero'
 alias restic='oc -n openshift-adp exec deployment/velero -c velero -it -- /usr/bin/restic'
 ```
 
-#### Enable Velero Shell completion
+#### Enable OADP CLI Shell completion
 
 ##### Bash
 Linux:
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)